cmd/repo-updater/repoupdater/server.go

```diff
@@ -412,15 +412,34 @@ func (s *Server) handleExternalServiceNamespaces(w http.ResponseWriter, r *http.
 		return
 	}
 
-	externalSvc := &types.ExternalService{
-		Kind:   req.Kind,
-		Config: extsvc.NewUnencryptedConfig(req.Config),
+	var (
+		externalSvc *types.ExternalService
+		err         error
+		result      *protocol.ExternalServiceNamespacesResult
+	)
+
+	externalServiceID := req.ExternalServiceID
+
+	if externalServiceID != nil {
```

Comment:
Nit: The newline could be removed here, since the variable assignment and the if check is related to each other - it may make more sense to cluster them together.

```suggestion
	externalServiceID := req.ExternalServiceID
	if externalServiceID != nil {
```

Reply:
updated